klippy/extras/gcode_move.py

```diff
@@ -256,8 +256,9 @@ def cmd_GET_POSITION(self, gcmd):
         kin_pos = " ".join(["%s:%.6f" % (a, v) for a, v in kinfo])
         toolhead_pos = " ".join(["%s:%.6f" % (a, v) for a, v in zip(
             "XYZE", toolhead.get_position())])
-        gcode_pos = " ".join(["%s:%.6f"  % (a, v)
-                              for a, v in zip("XYZE", self.last_position)])
+        gcode_pos = " ".join(
+                 ["%s:%.6f"  % (a, v)
+                 for a, v in zip("XYZE", self._get_gcode_position())])
```

Comment:
I've noticed that most of the Klipper code aligns the arguments to the opening parenthesis when they spillover like this, so I formatted it this way when I tested it out this morning:

```
         gcode_pos = " ".join(["%s:%.6f"  % (a, v)
                              for a, v in zip("XYZE",
                                              self._get_gcode_position())])
```